assets/prefabs/animals/animalAdjectives.prefab

```diff
@@ -0,0 +1,143 @@
+{
+  "TownNameAffix": {
```

Comment:
It's the name of the existing Prefix/Postfix system; it has nothing particularly unique to towns, as far as I can ascertain. It could do with a refactoring.